### PR TITLE
[Mellanox] Add copyright headers to Mellanox-SN4600C-D100C12S2 SKU

### PR DESCRIPTION
**Pull request by alexrallen**
#### Why I did it

Missing NVIDIA copyright header from some files.

#### How I did it

Added copyright header

#### How to verify it

N/A

#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [ ] 202012
- [ ] 202106

#### Description for the changelog
[mellanox] Add copyright headers to Mellanox-SN4600C-D100C12S2 SKU

### CODE REVIEW

File: device/mellanox/x86_64-mlnx_msn4600c-r0/Mellanox-SN4600C-D100C12S2/buffers_defaults_t0.j2 [hunk 1]
```diff
@@ -1,3 +1,20 @@
+{#
+    Copyright (c) 2018-2021 NVIDIA CORPORATION & AFFILIATES.
```
Comment (liat-grozovik) [inline device/mellanox/x86_64-mlnx_msn4600c-r0/Mellanox-SN4600C-D100C12S2/buffers_defaults_t0.j2:2]: is this file from 201811? if not the y-y should be from file creation to today

File: device/mellanox/x86_64-mlnx_msn4600c-r0/Mellanox-SN4600C-D100C12S2/buffers_defaults_t1.j2 [hunk 2]
```diff
@@ -1,3 +1,20 @@
+{#
+    Copyright (c) 2018-2021 NVIDIA CORPORATION & AFFILIATES.
```
Comment (liat-grozovik) [inline device/mellanox/x86_64-mlnx_msn4600c-r0/Mellanox-SN4600C-D100C12S2/buffers_defaults_t1.j2:2]: same as above

**Comment by alexrallen** [comment]
@liat-grozovik please take a look and approve. Thanks.

**Review by liat-grozovik — APPROVED** [review]

**Pull request merged**